git-updater/templates/update.sh.j2

```diff
@@ -22,6 +22,7 @@ log "## updating ${GIT_PATH}" "${UPDATE_LOG} ${GIT_LOG}"
 cd ${GIT_PATH}
 git pull origin master 2>> "$ERROR_LOG" | tee -a ${GIT_LOG}
 git submodule update 2>> "$ERROR_LOG" | tee -a ${GIT_LOG}
+git remote prune origin && git repack && git prune-packed && git reflog expire --expire=1.month.ago && git gc --aggressive
```

Comment:
I think this is not what we want, or at least not here. When deployed to the virtual machine it seems to still be filling up (currently kind of stable at 55%), however in addition to that the CPU:s are now maxed out doing nothing but repacking.

![image](https://user-images.githubusercontent.com/352149/129039685-efb87538-f0e6-48b3-837f-92f5aefac815.png)


Ways forward that I could see are

- Run this cleanup thing less frequently, ie add another update script that runs.
- Figure out what parts of this command have the most impact and trim it down, do we really need `--expire=1.month.ago`

Reply:
Little update it went up to 60% now down to 50% again so it seems to be doing something.

Reply:
Yeah, I also found that `git gc` didn’t actually help with the issue on the wb-reconcile system (before it was reset) – it seemed like it filled up the disk with new pack files and then died (out of disk space) before it got a chance to delete old stuff.

Do we have any clue why the repeated `git pull`s fill up the disk?

Reply:
I've checked that the cloud computer has 8 cores and 16GB of memory instead of 2 puny cores on VM. Would it be able to handle the workload? Is it worth a try?
anw, the `--expire=1.month.ago` means that all refs that are older than 1 month will be deleted.